frontends/common/options.cpp

```diff
@@ -145,7 +145,7 @@ FILE* CompilerOptions::preprocess() {
 #else
         std::string cmd("cpp");
 #endif
-        cmd += cstring(" 2>/dev/null -undef -nostdinc -I") +
+        cmd += cstring(" -undef -nostdinc -I") +
```

Comment:
Since this won't change the fail/success of the preprocessor (just filter warning messages), it would seem we need to check for malformed strings (and emit an error) in either case.